sample.env

```diff
@@ -184,6 +184,21 @@ RELATIVE_URL_ROOT=/b
 #   recording: Sessions are recorded
 ROOM_FEATURES=mute-on-join,require-moderator-approval,anyone-can-start,all-join-moderator,recording
 
+# Specify which room join settings you would like the users to configure on join / enter room session
```

Comment:
I'm not sure we'll want this configurable at the ENV level. I would prefer to have them at the role level, though I'll need to spend some more time thinking about it

Reply:
Ah thanks, let us know if you want to move it on other level, we will change it

Reply:
Hi @farhatahmad , is there any update on this? Have you decide where we should put the configuration? Or maybe I misunderstand something?